css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+    content: url(.//img/user.svg);
+    position: absolute;
+    left: 0px;    
+}
+.button-compare {
+    padding-left: 20px;
+}
+.button-compare::before {   
+    content: url(.//img/compare.svg); 
+    position: absolute;  
+    left: 0px;   
+}
+.button-basket {
+    padding-left: 23px;
+}
+.button-basket::before {
+    content: url(.//img/cart.svg);
+    position: absolute;
+    left: 0px;  
+}
+.user-buttons a:hover,
+.nav-list a:hover {
+    opacity: 0.6;
+}
+.user-buttons a:active,
+.nav-list a:active {
+    opacity: 0.3;
+}
+.nav {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: flex-start;
+    width: auto;
+    margin-left: 60px;
+    margin-right: 60px;
+}
+.nav-list {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    width: 100%;
+}
+.nav-link {
+    font-size: 18px;
+    line-height: 24px;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.nav-catalog {
+    width: 50%;
+}
+.nav-delivery {
+    margin-right: 35px;
+    margin-left: 20px;
+}
+.nav-warranty {
+    margin-right: auto;
+}
+.nav-catalog-link {
+    position: relative;
+    padding-right: 28px;
+}
+.nav-catalog-link::after {
+    content: url(.//img/plus.svg);
+    position: absolute;
+    right: 0px;
+    top: 2px;
+    opacity: 0.3;    
+}
+
+.dropdown-menu {
+    display: none;
+    position: absolute;
+
+    min-width: 1160px;
+    padding-left: 60px;
+    padding-top: 40px;
+    padding-bottom: 20px;
+    left: -60px;
+    top: 21px;
+
+    background-color: #f7e296;
+    z-index: 2;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 15px;
+    line-height: 36px;
+}
+.dropdown-menu div {
+    display: flex;
+    flex-direction: column;
+    margin-right: 40px;
+}
+.nav-catalog-link:hover ~ .dropdown-menu {
+    display: flex;
+}
+.dropdown-menu:hover {
+    display: flex;
+}
+
+h1 {
+    display: flex;
+    max-width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-left: 60px;
+    font-size: 47px;
+    line-height: 48px;
+}
+
+/* BREADCRUMBS */
+
+.breadcrumbs {
+    display: flex;
+    max-width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 48px;
+    padding-left: 60px;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 14px;
+    line-height: 24px;
+    
+}
+.breadcrumbs a {
+    position: relative;
+    
+}
+.breadcrumbs li {
+    padding-right: 40px;
+    opacity: 0.3;
+}
+.breadcrumbs li:hover {
+    opacity: 0.6;
+}
+.breadcrumbs li:active {
+    opacity: 0.1;
+}
+.breadcrumbs a::after {
+    content: url(.//img/nav-arrow.svg);
+    position: absolute;
+    right: -20px;
+}
+
+/* SECTION FILTER */
+.section-filter {
+    display: flex;
+    position: relative;
+    justify-content: flex-start;
+    max-width: 1440px;
+    margin-left: auto;
+    margin-right: auto;
+    
+}
+.section-filter aside {
+    order: -1;
+}
+.container {
+    padding-left: 72px;
+}
+.sorting {
```

Comment:
in my projects those background of Filter and Sortment are pseudo elements of main containers

<pre>
  .catalogue::before {
  position: absolute;
  content: "";
  top: 0;
  left: 0;
  right: 0;
  height: 70px;
  z-index: 0;
  background-color: #ebebeb;
}
</pre>

Reply:
Revised approach and worked with paddings instead of margins. Now this point is done :)